src/components/WebexAddCollaborators/WebexAddCollaborators.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import {InputField} from '../generic';
+
+/**
+ * Webex Add Collaborators component
+ *
+ */
+export default function WebexAddCollaborators() {
+  return (
+    <div>
+      <InputField
+        placeholder="Add people"
```

Comment:
are we sure we should be hardcoding strings? We should probably invest time in properly setting up translations and use them from the start.

Reply:
For now there is no translation setup in the repo.We will confirm with lalli's team before setting it up.